### PR TITLE
[5.0] Fix doubled Toolbar for modal views

### PR DESCRIPTION
**Pull request by Fedik**
Pull Request for Issue #41572, #41578 .

### Summary of Changes

The patch renders toolbar in tmpl/modal.php layout, instead of in global component.php

### Testing Instructions
Edit menu for Single Article, use modal select, create, edit for article.
Aslo try menu for Category, Contact, Newsfeed.

Edit an article, select an image in Media. Repeat it in front-end.

Also try to add a module to dashboard.

### Actual result BEFORE applying this Pull Request
There a doubled toolbars.

### Expected result AFTER applying this Pull Request
No doubled toolbar.

### Link to documentations
Please select:
- [ ] Documentation link for docs.joomla.org: <link>
- [x] No documentation changes for docs.joomla.org needed
- [ ] Pull Request link for manual.joomla.org: <link>
- [x] No documentation changes for manual.joomla.org needed

Reference:
 - https://github.com/joomla/joomla-cms/pull/40462

### CODE REVIEW

File: administrator/components/com_media/tmpl/file/default.php [hunk 1]
```diff
@@ -63,6 +56,11 @@
 
 $this->useCoreUI = true;
 ?>
+<?php if ($tmpl === 'component'): ?>
```
Comment (richard67) [inline administrator/components/com_media/tmpl/file/default.php]:
```suggestion
<?php if ($tmpl === 'component') : ?>
```
PHPCS

File: administrator/components/com_media/tmpl/media/default.php [hunk 2]
```diff
@@ -63,4 +56,9 @@
 ];
 $this->document->addScriptOptions('com_media', $config);
 ?>
+<?php if ($tmpl === 'component'): ?>
```
Comment (richard67) [inline administrator/components/com_media/tmpl/media/default.php]:
```suggestion
<?php if ($tmpl === 'component') : ?>
```
PHPCS

File: administrator/components/com_media/tmpl/file/default.php [hunk 3]
```diff
@@ -36,13 +36,6 @@
 
 $input->set('hidemainmenu', true);
 
-// Load the toolbar when we are in an iframe
-if ($tmpl == 'component') {
-    echo '<div class="subhead noshadow">';
-    echo Toolbar::getInstance('toolbar')->render();
```
Comment (Quy) [inline administrator/components/com_media/tmpl/file/default.php:42]: remove import statement 😄

Comment (Fedik) [reply administrator/components/com_media/tmpl/file/default.php:42]: Ups :smile:

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on 2a6f1be5567f54e888bb16799a1f459df56159cb<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/41600">issues.joomla.org/tracker/joomla-cms/41600</a>.</sub>

**Comment by Quy** [comment]
For Create Article, buttons should be in the footer.

41600-header

like the following:

41600-footer

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 2a6f1be5567f54e888bb16799a1f459df56159cb<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/41600">issues.joomla.org/tracker/joomla-cms/41600</a>.</sub>

**Comment by HLeithner** [comment]
thanks

**Pull request merged**